### PR TITLE
Added make to Vagrantfile and added build instructions to README.md

### PR DESCRIPTION
**Pull request by Sag0Sag0**

### CODE REVIEW

**Comment by gandaldf** [comment]
Thank you! 
Using these instruction I was able to build and run the os with no problem!
I would add this for Mac OS users:

Install QEMU (using [Brew](https://brew.sh/index_it.html)):
```
$ brew install qemu
```

Install Vagrant (using [Brew](https://brew.sh/index_it.html)):
```
$ brew cask install virtualbox
$ brew cask install vagrant
$ brew cask install vagrant-manager
```

**Comment by joeblew99** [comment]
Please add this to readme :)

**Comment by Sag0Sag0** [comment]
I have added the homebrew description for OSX.

File: README.md [hunk 1]
```diff
@@ -4,3 +4,21 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/achilleasa/gopher-os)](https://goreportcard.com/report/github.com/achilleasa/gopher-os)
 
 Let's write an experimental OS in Go!
+
+## Building
+
+There are several dependancies you need to install for this to work.
```
Comment (ilyaglow) [inline README.md]: dependancies -> depend**e**ncies. Same for line 14 and 21

Comment (Sag0Sag0) [inline README.md]: Thanks.

File: Vagrantfile [hunk 2]
```diff
@@ -17,7 +17,7 @@ Vagrant.configure("2") do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     apt-get update
-    apt-get install -y nasm gccgo xorriso
+    apt-get install -y nasm gccgo xorriso make
```
Comment (achilleasa) [inline Vagrantfile]: I thought `make` was already part of the xenial image. Did you encounter issues with `make` not being present?

Comment (Sag0Sag0) [inline Vagrantfile]: Yes

**Comment by achilleasa** [comment]
Closing as #49 adds more detailed documentation. Feel free to rebase and reopen if you think that the version on master is missing some additional bits.

**Pull request closed without merging**